File: farm/farm/src/main/java/com/glauco/farm/api/dto/ProducaoFazendaDTO.java
package com.glauco.farm.api.dto;

import java.util.List;

import com.glauco.farm.api.model.Fazenda;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ProducaoFazendaDTO {

    private Fazenda fazenda;

    private Integer ano;

    private Double valor;

    private List<ProducaoTalhaoDTO> producaoTalhaoDTOList;

}

File: farm/farm-front/src/app/dto/producao.fazenda.dto.ts
import {Fazenda} from '../model/fazenda/fazenda.model';
import {ProducaoTalhaoDto} from './producao.talhao.dto';

export class ProducaoFazendaDto {
  public fazenda: Fazenda;
  public ano: number;
  public valor: number;
  public producaoTalhaoDTOList: Array<ProducaoTalhaoDto>;
}
